packages/expo-firebase/src/analytics/FirebaseAnalytics.js

```diff
@@ -0,0 +1 @@
+import 'expo-firebase-analytics';
```

Comment:
Shouldn’t this export something too or does it implicitly modify a global?

Reply:
modify, in the next version it'll export all the types